JoyfillExample/JoyfillExample/Views/Documents/DocumentJoyDoc.swift

```diff
@@ -136,11 +136,7 @@ class ViewController: UIViewController, onChange, UIImagePickerControllerDelegat
     func convertImageToDataURI(uri: UIImage) {
         if let imageData = uri.jpegData(compressionQuality: 1.0) {
             let base64String = imageData.base64EncodedString()
-            updateImage = true
-            uploadedImageCount[imageIndexNo] = ["data:image/jpeg;base64,\(base64String)"]
-            uploadedSingleImage[imageIndexNo] = ["data:image/jpeg;base64,\(base64String)"]
-            uploadedMultipleImage[imageIndexNo].append("data:image/jpeg;base64,\(base64String)")
-            joyDoc.reloadData()
+            onUploadAsync(imageUrl: "data:image/jpeg;base64,\(base64String)")
```

Comment:
So what does this do? Why would it need to be here in the example project and why would a dev using our project that just needs the array of uploaded base64 and existing url images need this?

Remember that we only handle 2 scenarios:
1. When they upload brand new images and the developer onUploadAsync receives an array of base64 images
2. When they upload more images to a set of existing images in an image field or image table cell field and then they would receive an array of the existing fields images values which could be urls they saved or base64 and the new uploaded in addition images that would be base64.

The developer just wants an array after images are upload. onUploadAsync just acts same as the handlers "handleOnChange", "handleOnFocus" ect...

Reply:
Hello @jeremypagley 

Let me explain the flow, what was discussed with Josh regarding image upload
1. Image upload feature will be handle by the developer.
2. Once user will upload the image and click on save button then they will call our joyfill api to upload the images and then we return all the images for that particular user
3. suppose user upload first image then they will see the image link in URL. if user do have existing images, then will upload another images then array will contain all the existing and new image

Regarding onUploadAsync, this is what we have used in our test app to received the uploaded image URL. same thing will be done by developer while implement the image picture.

Reply:
this still is not making sense? In my comment I said that when a user uploads images to an image field they should receive a list of the uploaded images in a function just like handleOnChange function callback.

See below:
![Screenshot 2023-12-19 at 11 07 25 AM](https://github.com/joyfill/components-ios/assets/5873346/9082de10-0bae-4900-83b9-a4c85e6b2c69)

Reply:
Done